libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -158,14 +158,22 @@ void AP_CRSF_Telem::process_rf_mode_changes()
     if (uart == nullptr) {
         return;
     }
-    // warn the user if their setup is sub-optimal
-    if (_telem_last_report_ms == 0 && !uart->is_dma_enabled()) {
-        gcs().send_text(MAV_SEVERITY_WARNING, "CRSF: running on non-DMA serial port");
-    }
-    // report a change in RF mode or a chnage of more than 10Hz if we haven't done so in the last 5s
+    // report a change in RF mode or a change of more than 10Hz if we haven't done so in the last 5s
     if ((now - _telem_last_report_ms > 5000) &&
         (_telem_rf_mode != current_rf_mode || abs(int16_t(_telem_last_avg_rate) - int16_t(_scheduler.avg_packet_rate)) > 25)) {
         if (!rc().suppress_crsf_message()) {
+            // warn the user if their setup is sub-optimal
```

Comment:
the way it is now users will get the message only when there's an RF mode change or rate change, is this intentional?

Reply:
I think we always get at least one mode change - currently I see a lot of spam at startup before the mode change has been detected